insights/Extract_MC_Insights_DBX.py

```diff
@@ -0,0 +1,70 @@
+# Databricks notebook source
+# MAGIC %md # Environment Setup
+
+# COMMAND ----------
+
+#Install the Monte Carlo Python Library (Notebook scoped)
+#More info here: https://docs.databricks.com/libraries/notebooks-python-libraries.html#install-a-library-with-pip
+%pip install pycarlo
+
+# COMMAND ----------
+
+# DBTITLE 1,Runtime Variables
+# Monte Carlo Credentials stored in DBX Secret Key Repo called "monte-carlo-creds":
+mcd_id = dbutils.secrets.get(scope = "monte-carlo-creds", key = "mc-id")
+mcd_token = dbutils.secrets.get(scope = "monte-carlo-creds", key = "mc-token")
+
+# Full List of Reports Needed, Edit as Needed for Databricks
+insight_names = ["monitors","cleanup_suggestions","events","incident_history"]
+insight_report_names = ["monitors.csv","cleanup_suggestions.csv","events.csv","incident_history.csv"]
+
+# Other variables which you should customize:
+mcd_profile=""
+table_schema = 'com_us_alyt_04'
+user = 'gordon.strodel@takeda.com'
+
+# COMMAND ----------
+
+# MAGIC %md # Load Insights to DBX
+
+# COMMAND ----------
+
+from pycarlo.core import Client, Query, Mutation, Session
+import requests
+from pyspark.sql.functions import *
+import io
+import pandas as pd
+from datetime import *
+
+client = Client(session=Session(mcd_id=mcd_id, mcd_token=mcd_token,mcd_profile=mcd_profile))
+today = datetime.today()
+
+for i in range(len(insight_report_names)):
```

Comment:
Nit - Can use [zip](https://docs.python.org/3/library/functions.html#zip) for this too